### PR TITLE
Improvement/seperate init

### PR DESCRIPTION
**Pull request by aboubakar14**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -16,4 +16,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  - Add new Ollama model files (for generate and message)
  - Handle new /api/message endpoint for Ollama, that send messages to the Ollama AI and get a response with the associated context
  - Added new Gemini model files (for generate and message). Conversation with a context is not supported.
+ - The initilize script is now separated from the main script. It can be launched from the root folder, anytime.
```
Comment (Zorin95670) [inline changelog.md]: typo

File: README.md [hunk 2]
```diff
@@ -67,9 +67,10 @@ First install pipenv and create a virtual environment with the needed packages:
 
     To deactivate your virtual environment: ``` deactivate ```
 
-Then launch the API using hypercorn (from the root folder):
+Then launch the API using hypercorn (from the root folder and your virtual environment is activated):
 
 ```sh
+python3 initialize.py
```
Comment (Zorin95670) [inline README.md]: add explanation on the initiajize script before

File: changelog.md (hunk 1 above)
Comment (aboubakar14) [reply changelog.md]: done

File: README.md (hunk 2 above)
Comment (aboubakar14) [reply README.md]: done

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**